packages/cli/src/commands/epochs/send-validator-payment.ts

```diff
@@ -0,0 +1,43 @@
+import { BaseCommand } from '../../base'
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { CustomFlags } from '../../utils/command'
+
+export default class SendValidatorPayment extends BaseCommand {
+  static description =
+    'Sends the allocated epoch payment to a validator, their group, and delegation beneficiary.'
+
```

Comment:
how about adding an alias "validator:send-payment" 

i suspect people will look for a command related to validators there